.changelog/19544.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+ui: Replaced single-line variable value fields with multi-line textareas and updated the components to use newer styles, generally
```

Comment:
nit: "the components" refers specifically to those on variables pages only, right?  I think that vague wording and the "generally" on the end makes me think it might be broader than that.  but, I don't want to suggest being toooo wordy, because that could detract from the main feature here: multi-line variable text boxes.

could maybe make two separate \`\`\`release-note:...\`\`\` entries for the two change categories?

Reply:
Updated to multiple entries, thanks!